Gaijinizer patch/Map402.txt

```diff
@@ -2,512 +2,624 @@
+that we're the same size...
 
+Before Ena-chan gets back.#Ron
 > END STRING
 
 > BEGIN STRING
 ぅ……あぁ………♡
-> CONTEXT: Map402/events/1/pages/0/143/Dialogue/kazuya-0 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/143/Dialogue/kazuya-0
+U-Uaah... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]は、マイに優しく抱きしめられた……
-> CONTEXT: Map402/events/1/pages/0/147/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/147/Dialogue
+\\N[1] has been gently embraced by Mai...#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]も、マイの腰に手を回そうとするが
 
 上半身をすっぽり、おっぱいに包まれている……
-> CONTEXT: Map402/events/1/pages/0/149/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/149/Dialogue
+\\N[1] tries to wrap his arms around Mai's waist but
+his upper half has been snugly wrapped in her breasts...#Ron
 > END STRING
 
 > BEGIN STRING
 ご、ごめんね。こんなおっぱいで……
 
 私………昔からこんな不格好な体格で……
 胸だけ小さくしようとしても、上手くいかないんだ…
-> CONTEXT: Map402/events/1/pages/0/153/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/153/Dialogue/enemy27
+S-Sorry, my boobs are getting in the way.
+My body's... always been awkward and clumsy...
+I've tried shrinking just my chest but it's never
+gone well.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 エナちゃんは、自分の太ももと同じで
 りっぱな『個性』だって言ってくれたけど……
 
 私は不便だし、邪魔だし、好きじゃない…
-> CONTEXT: Map402/events/1/pages/0/158/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/158/Dialogue/enemy27
+Ena-chan said it adds character in the same way
+her thighs do, but they're so inconvenient,
+always getting in the way... I don't like them.#Ron
 > END STRING
 
 > BEGIN STRING
 で、でも、そんな自分の身体が大好きになる時間がね
 
 \\N[1]くんみたいに、触ったニンゲンが
 すごく情けない顔で、気持ちよくなってる時なんだ♡
-> CONTEXT: Map402/events/1/pages/0/164/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/164/Dialogue/enemy27
+B-But there are times I love them, like when a
+human such as you gets so much pleasure touching
+them that they make a really pathetic face.#Ron
 > END STRING
 
 > BEGIN STRING
 だから……ね？♡\\N[1]くん♡
 
 我慢なんかせずに、いっぱい感じて……♡
 たくさんたくさん、愛してあげるから………♡♡
-> CONTEXT: Map402/events/1/pages/0/169/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/169/Dialogue/enemy27
+So... ♡\\N[1]-kun... ♡
 
+Don't hold back, feel my boobs all you want... ♡
+And I'll give you lots and lots of love, okay? ♡♡#Ron
 > END STRING
 
 > BEGIN STRING
 あぁ………♡
-> CONTEXT: Map402/events/1/pages/0/179/Dialogue/kazuya-0 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/179/Dialogue/kazuya-0
+O-Okay... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 ほら………♡
 
 もっとたくさん、ぎゅ～ってしてあげる……♡♡
-> CONTEXT: Map402/events/1/pages/0/181/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/181/Dialogue/enemy27
+Come on... ♡
 
+I'll give you even more squeezes... ♡♡#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]は、マイのおっぱいに身体を解され
 
 心身ともども、骨抜きになってしまった……
-> CONTEXT: Map402/events/1/pages/0/187/Dialogue < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/187/Dialogue
+\\N[1] body is disentangled from Mai's breasts...
```

Comment:
"\\N[1] body" -> "\\N[1]'s body"
It would most likely go off-screen depending on the player's name, though.